internal/service/instance/nginx_instance_service.go

```diff
@@ -202,29 +201,15 @@ func parseNginxVersionCommandOutput(ctx context.Context, output *bytes.Buffer) *
 	return nginxInfo
 }
 
-func parseNginxVersion(line string) (version, plusVersion string) {
-	ossVersionMatches := ossVersionRegex.FindStringSubmatch(line)
-	plusVersionMatches := plusVersionRegex.FindStringSubmatch(line)
+func parseNginxVersion(line string) string {
+	ossVersionMatches := ossVersionRegex.FindString(line)
```

Comment:
I think we can actually simplify this further by using just one regex for both oss and plus. Something like this will work for both I think:
```
(?P<name>\S+)\/(?P<version>.*)
```